openjdk/build.xml

```diff
@@ -47,7 +47,7 @@
 							<property name="jtregTar" value="jtreg_6_1_1"/>
 						</then>
 						<else>
-							<property name="jtregTar" value="jtreg_7_1"/>
+							<property name="jtregTar" value="jtreg_7_1_1_1"/>
```

Comment:
Does this mean jtreg_7_1 can be replaced by jtreg_7_1_1_1? So we can do a cleanup with jtreg_7_1? @luhenry

Reply:
Once this PR has been merged I expect we can remove `jtreg_7_1` at https://github.com/adoptium/TKG/pull/381/files#diff-6548ada249042f87589ac5683c2f76bd9b8447695fd19fd02ad3030a2592b775R146-R152 and https://github.com/adoptium/ci-jenkins-pipelines/pull/532/files#diff-a0facb89f6e82d0bdeb77e87b5306d0a9d185eef333ba542dd83cee60afb885cR128